Wikipedia/Code/WebViewController.m

```diff
@@ -150,53 +150,54 @@ - (void)userContentController:(WKUserContentController*)userContentController di
                 // A standard external link, either explicitly http(s) or left protocol-relative on web meaning http(s)
                 if ([href hasPrefix:@"#"]) {
                     [self scrollToFragment:[href substringFromIndex:1]];
-                } else if ([href hasPrefix:@"//"]) {
-                    // Expand protocol-relative link to https -- secure by default!
-                    href = [@"https:" stringByAppendingString:href];
-                }
-                NSURL* url = [NSURL URLWithString:href];
-                NSCAssert(url, @"Failed to from URL from link %@", href);
-                if (url) {
-                    [self wmf_openExternalUrl:url];
+                } else {
+                    if ([href hasPrefix:@"//"]) {
+                        // Expand protocol-relative link to https -- secure by default!
+                        href = [@"https:" stringByAppendingString:href];
+                    }
+                    NSURL* url = [NSURL URLWithString:href];
+                    NSCAssert(url, @"Failed to from URL from link %@", href);
+                    if (url) {
+                        [self wmf_openExternalUrl:url];
+                    }
                 }
             }
-        } else if (message.body[@"imageClicked"]) {
-            NSDictionary* imageClicked = message.body[@"imageClicked"];
-            WMFImageTag* imageTagClicked = [[WMFImageTag alloc] initWithSrc:imageClicked[@"src"]
-                                                                     srcset:nil
-                                                                        alt:nil
-                                                                      width:imageClicked[@"width"]
-                                                                     height:imageClicked[@"height"]
-                                                              dataFileWidth:imageClicked[@"data-file-width"]
-                                                             dataFileHeight:imageClicked[@"data-file-height"]];
-
-            if (![imageTagClicked isSizeLargeEnoughForGalleryInclusion]) {
-                return;
-            }
+        }
+    } else if (message.body[@"imageClicked"]) {
```

Comment:
@coreyfloyd this was the problem - this needs to be nested inside of the `} else if ([message.name isEqualToString:@"clicks"]) {` if statement, not at same level.